client/client.go

```diff
@@ -233,3 +245,41 @@ func (c *Client) loadShortID() error {
 	c.shortID = binary.LittleEndian.Uint32(data)
 	return nil
 }
+
+// readCTFile looks for a CT settings file, and if it is found,
+// parses it by line for the energy multiplier and divider.
+func (c *Client) readCTSettingsFile() error {
+	path := filepath.Join(c.staticBaseDir, CTSettingsFile)
+	file, err := os.Open(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil // use defaults
+		} else {
+			return fmt.Errorf("error opening ct settings file: %v", err)
+		}
+	}
+	defer file.Close()
+	var mult, div float64
+	scanner := bufio.NewScanner(file)
```

Comment:
Let's read the whole file into memory at once, just like elsewhere in the client.